com.unity.multiplayer.mlapi/Runtime/Messaging/InternalMessageHandler.cs

```diff
@@ -408,7 +407,7 @@ public void HandleNamedMessage(ulong clientId, Stream stream)
         {
             PerformanceDataManager.Increment(ProfilerConstants.NamedMessageReceived);
             ProfilerStatManager.NamedMessage.Record();
-            
+
```

Comment:
Please leave these types of changes out since it's just whitespace, I'm guessing you modified this and then undid the changes later but no need to touch the file with this

Reply:
It was a mistake. I'll revert that.